helm/templates/deployment.yaml

```diff
@@ -70,7 +86,7 @@ spec:
             {{- end }}            
           envFrom:
             - configMapRef:
-                name: {{ printf "%s-configmap" (include "ts-server-boilerplate.fullname" .) }}
+                name: {{ printf "%s-configmap" (include "overseer.fullname" .) }}
```

Comment:
use the template that you defined to configmap name:
`{{ $configmapName }}`

Reply:
don't forget 'include' at the top of the file

Reply:
done